metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema.py

```diff
@@ -377,7 +377,11 @@ def _make_bigquery_view(view: bigquery.Row) -> BigqueryView:
             materialized=view.table_type == BigqueryTableType.MATERIALIZED_VIEW,
             size_in_bytes=view.get("size_bytes"),
             rows_count=view.get("row_count"),
-            labels=parse_labels(view.labels) if hasattr(view, "labels") else None,
+            labels=(
+                parse_labels(view.labels)
+                if hasattr(view, "labels") and isinstance(view.labels, str)
```

Comment:
when would labels not be None or string?

```suggestion
                if view.get("labels")
```